Gems/ROS2/Code/Include/ROS2/RobotImporter/SDFormatSensorImporterHook.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/Entity.h>
+#include <AzCore/std/containers/unordered_set.h>
+#include <AzCore/std/function/function_template.h>
+#include <AzCore/std/string/string.h>
+
+namespace sdf
+{
+    inline namespace v13
+    {
+        enum class SensorType;
+        class Sensor;
+    } // namespace v13
+} // namespace sdf
+
+namespace ROS2::SDFormat
+{
+    //! Hook used in ROS2 Gem Robot Importer to create ROS2 sensor components based on SDF model description file.
+    //! See the <sensor> element at http://sdformat.org/spec?ver=1.10&elem=sensor for more details.
+    struct SensorImporterHook
+    {
+        AZ_TYPE_INFO(SensorImporterHook, "{23f189e3-8da4-4498-9b89-1ef6c900940a}");
+
+        //! Types of the sensor in the SDF file that can be reused in O3DE.
```

Comment:
What does it mean reused? Perhaps add a comment about equivalence or similarity or mapping to the struct top level doxy comment.

Reply:
Renamed to terms such as `mapping` and `supported`